com.obsproject.Studio.Plugin.DistroAV.metainfo.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="addon">
+  <id>com.obsproject.Studio.Plugin.DistroAV</id>
+  <provides>
+    <id>com.obsproject.Studio.Plugin.NDI</id>
+  </provides>
+  <replaces>
+    <id>com.obsproject.Studio.Plugin.NDI</id>
+  </replaces>
+  <extends>com.obsproject.Studio</extends>
+  <name>DistroAV</name>
+  <developer>Paul Peavyhouse</developer>
```

Comment:
distroav@distroav.org

Reply:
A mail can't be used here, it needs to be legible.

But I replaced the author by DistroAV, newer metainfo rules that I forgot to apply ask for an id (usually a reverse DNS).
https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-developer